utils/node_local_test.py

```diff
@@ -983,24 +984,24 @@ def run_il_test(server, conf):
 
     pools = get_pool_list()
 
-    # TODO: This doesn't work with two pools, there appears to be a bug
-    # relating to re-using container uuids across pools.
+    # TODO: This doesn't work with two pools, partly related to
```

Comment:
(style)  'doesn'' may be misspelled - perhaps 'doesn't'?
(lint) This doesn't work with two pools, partly related to